vmaas/reposcan/exporter.py

```diff
@@ -748,6 +749,47 @@ def _dump_oval(self, dump):
                 dump.execute("insert into oval_module_test_detail values (?, ?)",
                              (oval_module_test_id, module_stream))
 
+    def _dump_csaf(self, dump):
+        """Select CSAF data"""
+        # Dump data from csaf_product table
+        dump.execute("""CREATE TABLE IF NOT EXISTS csaf_product (
+                        id INTEGER PRIMARY KEY AUTOINCREMENT,
+                        cpe_id INTEGER NOT NULL,
+                        package_name_id INTEGER DEFAULT 0,
+                        package_id INTEGER DEFAULT 0,
+                        module_stream TEXT DEFAULT ''
```

Comment:
these defaults and coalesce are because sql.Scan in go can't load NULL to int/string? I think we did it for OVAL the other way around, added coalesce to select in go code. Both solutions are fine with me.

Reply:
Yes. It returned ~`can't parse NULL to int` error without it. For me it is no difference either.